config.yaml

```diff
@@ -51,7 +52,7 @@ trainer:
   profiler: null
   detect_anomaly: false
   barebones: false
-  plugins: null
+  plugins: ddp_sharded
```

Comment:
This is not a valid choice in Lightning 2.1 (I assume you have this version since it is written in the comment in this file). 

The distributed strategy is no longer chosen with the "plugins" argument. There is now a separate `strategy` argument for this. For example:

```py
Trainer(strategy="ddp")
```

Furthermore, ddp_sharded was replaced by fsdp, so you probably want to pass
```
strategy: "fsdp" 
```

here, or configure the FSDP strategy further. Tutorial for further reference: https://lightning.ai/docs/pytorch/stable/advanced/model_parallel/fsdp.html#fully-sharded-training


I'm commenting here because I saw the issue you posted over at Lightning. 
Cheers